src/main/java/io/github/communitymod/core/util/enums/ModToolMaterials.java

```diff
@@ -0,0 +1,55 @@
+package io.github.communitymod.core.util.enums;
+
+import java.util.function.Supplier;
+
+import io.github.communitymod.core.init.ItemInit;
+import net.minecraft.util.LazyLoadedValue;
+import net.minecraft.world.item.Tier;
+import net.minecraft.world.item.crafting.Ingredient;
+
+@SuppressWarnings("deprecation")
+public enum ModToolMaterials implements Tier {
+	   BEAN(0, 118, 2.0F, 0.0F, 15, () -> {
```

Comment:
You can remove the body of this lambda. It's just 1 line. `() -> Ingredient.of(ItemInit.BEANS.get())` works just fine!